.github/workflows/_test_t5x.yaml

```diff
@@ -268,6 +268,36 @@ jobs:
           name: ${{ steps.meta.outputs.JOB_NAME }}
           path: output/*
 
+  metrics:
+    needs: [multi-gpu-multi-node, single-process-multi-device]
+    runs-on: ubuntu-22.04
+
+    steps:
+      - name: Check out the repository under ${GITHUB_WORKSPACE}
+        uses: actions/checkout@v3
+
+      - name: Download artifacts
+        uses: actions/download-artifact@v3
+
+      - name: Run pytest
+        shell: bash -x {0}
+        run: |
+          pip install pytest pytest-reportlog tensorboard
+          for i in ${GITHUB_RUN_ID}-*; do
+            SUBDIR=$(echo $i | cut -d'-' -f2)
+            mv $i/$SUBDIR* .
+            python3 .github/workflows/baselines/summarize_metrics.py $SUBDIR --perf_summary_name "timing/steps_per_second" # create result json in baseline format
+          done
+
+          echo '## T5X MGMN Test Metrics' >> $GITHUB_STEP_SUMMARY
+          for i in *_metrics.json; do
+            echo $i | cut -d'.' -f1
+            echo '```json'
+            jq . $i
+            echo '```'
+          done | tee -a $GITHUB_STEP_SUMMARY
```

Comment:
Why doesn't t5x also run pytest like the pax tests do?

https://github.com/NVIDIA/JAX-Toolbox/blob/main/.github/workflows/_test_pax.yaml#L187C11-L187C121

Reply:
I was going to add the tests in a separate PR. I guess I might as well add it here now that we have all the baselines.

Reply:
Yea, let's add it in this PR since you may want to change the baselines if the tests don't pass according to your expectations